share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -37,10 +37,9 @@ macro(rocm_create_package)
     #Debs use underscrore between OS and architecture
     elseif(_os_id_ubuntu)
         STRING(CONCAT _SYSTEM_NAME ${_os_id} "-" ${_version_id} "_amd64")
-    elseif(_os_id_sles)
-        STRING(CONCAT _SYSTEM_NAME ${_os_id} "-" ${_version_id} ".amd64")
     else()
-        set(_SYSTEM_NAME ${CPACK_SYSTEM_NAME})
+    #For SLES and unsupported OSes
+        STRING(CONCAT _SYSTEM_NAME ${_os_id} "-" ${_version_id} ".amd64")
     endif()
```

Comment:
All modern Linux distributions have /etc/os-release. What should our behaviour be if it does not exist?

Reply:
We should just use `CMAKE_SYSTEM_NAME` when ` /etc/os-release` is not found.

Reply:
Please do not merge yet. I fixed it to go with default behavior. I am going to test this out and see.